Cargo.toml

```diff
@@ -1,46 +1,46 @@
 [package]
 name = "rdkafka"
-version = "0.37.0"
+version = "0.40.0"
```

Comment:
Why the version bump to 0.40 instead of 0.38?

Reply:
Oops, mismash. That was leftover from the tests I was doing on the crate I uploaded based on the fork. I'll fix it.